tests/testdata/snapshots/deployment.yaml/default

```diff
@@ -0,0 +1,60 @@
+---
+# Source: surrealdb/templates/deployment.yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: testrelease-surrealdb
+  labels:
+    helm.sh/chart: surrealdb-0.3.4
```

Comment:
I'm slightly worried about the maintenance hassle of updating this every time there is a new Helm chart. Same for the version. These will be in every yaml file.

Reply:
Thanks @MichelHollands! Me too- and that's why I added the option to update all the snapshots with a command:

```
UPDATE_SNAPSHOT="*" go test -v ./tests
```

We want tests, but I don't think forcing everyone to write unit tests in Go, shell, or helm-unittest, or even writing integration tests using `helm test`.

Instead, we provide a single command to update snapshots to easily update the snapshot to see if a pull request has any unexpected changes. It should be easier than writing tests while catching easy mistakes that are hard to detect with other tests.